packages/modelviewer.dev/index.html

```diff
@@ -56,7 +56,7 @@ <h3 class="grouping-title grouping-title-new quick-start">Quick Start</h3>
 <script type="module" src="https://unpkg.com/@google/model-viewer/dist/model-viewer.js"></script>
 
 <!-- Use it like any other HTML element -->
-<model-viewer src="shared-assets/models/Astronaut.glb" alt="A 3D model of an astronaut" ar ar-modes="webxr scene-viewer quick-look" environment-image="neutral" auto-rotate camera-controls></model-viewer>
+<model-viewer src="shared-assets/models/Astronaut.glb" alt="A 3D model of an astronaut" ar ar-modes="webxr scene-viewer quick-look" environment-image="neutral" xr-environment auto-rotate camera-controls></model-viewer>
```

Comment:
Let's put this in a slightly less prominent location. How about here: https://modelviewer.dev/examples/augmentedreality/#ar

Reply:
Sounds good to me. Updated, thanks!